core/modules/commands/savelibrarytiddlers.js

```diff
@@ -66,7 +66,7 @@ Command.prototype.execute = function() {
 		$tw.utils.createFileDirectories(pathname);
 		fs.writeFileSync(pathname,JSON.stringify(tiddler),"utf8");
 		// Collect the skinny list data
-		var pluginTiddlers = $tw.utils.parseJSONSafe(tiddler.text),
+		var pluginTiddlers = $tw.Wiki.pluginInfoModules[tiddler.type].parse(new $tw.Tiddler(tiddler)),
```

Comment:
It seems you use the prototype `$tw.Wiki.` function instead of `$tw.wiki`, which is the wiki instance. -- Is that intentional. 
`$tw.Wiki` is used that way several times. Not sure if that's the right thing to do.

Reply:
hmmm. OK it seems to follow existing code. :/